HISTORY.md

```diff
@@ -1,3 +1,6 @@
+# v4.9.2
```

Comment:
Sorry for the conflict.

I know there's been conflicting advice about what should be in PRs, but I'd like to make the case right now that changes to version numbers and HISTORY.md should _not_ be in PRs. It guarantees conflicts if more than one PR is open at a time.

I much prefer the pattern of performing the version bump and history update on `master` after one or more PRs are merged. This is also a good time to recompile everything in case any build artefacts have fallen out of sync (compiled CSS, generated API json, etc).

 @joshsylvester thoughts?

Reply:
Resolved the conflict.
About the version, if we could create a job to auto update and commit should be better.

Reply:
We do have a gulp task for performing patch, minor and major version bumps. Commit for releasing updates shouldn't be automated - they need to be curated (meaningful changelog written, occasionally batching multiple related updated into a single release, etc).